builtins/amp-img.md

```diff
@@ -125,6 +125,10 @@ For the `<img>` tag in `HTML`, the `sizes` attribute is used in conjunction with
 
 See [Responsive images with srcset, sizes & heights](https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/art_direction) for usage of `sizes` and `srcset`.
 
+**img-sizes**
+
+Like `sizes` above, sets the [`sizes`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-sizes) attribute on the underlying `img`. Unlike `sizes`, this does not change the width of the `amp-img` and is only used by the browser for picking which source to use from the `srcset`.
```

Comment:
I think this is trying to say:

The `img-sizes` attribute is used by the browser to pick up which source to from the `srcset`, unlike the above `sizes` attribute, it does not change with width of the `amp-img`.